### ISSUE TITLE
crash while setting hotkey

### ISSUE THREAD

**Issue by gmankab** [microsoft/terminal#11689]
### Windows Terminal version (or Windows build number)

10.0.22000.282, 1.11.2921.0

### Steps to reproduce

Settings, actions, add new, move tab backwards, Alt+Left, Yes

The video is displayed incorrectly on the github for me, so I uploaded it to YouTube
https://youtu.be/_sF7Cr2wHx0

https://user-images.githubusercontent.com/73340745/140421423-a00bcce7-09f1-44a7-83a7-bdd258c7e22a.mp4

### Expected Behavior

setting hotkey

### Actual Behavior

crash

**Comment by elsaco** [comment]
@gmankab if it's not inconvenient try cleaning `settings.json`, `state.json` and all the backup files in 

`C:\Users\your_user_name_here\AppData\Local\Packages\Microsoft.WindowsTerminal_8wekyb3d8bbwe\LocalState>`

WT 1.11.2921.0 kept crashing when adding/editing actions, sample log reading:

```
Index              : 73555
EntryType          : Error
InstanceId         : 1000
Message            : Faulting application name: WindowsTerminal.exe, version: 1.11.2110.19001, time stamp: 0x616f34a4
                     Faulting module name: Microsoft.Terminal.Settings.Editor.dll, version: 1.11.2110.19001, time stamp: 0x616f33d8
                     Exception code: 0xc000041d
                     Fault offset: 0x0000000000026514
                     Faulting process id: 0x308c
                     Faulting application start time: 0x01d7d34d8ddb259d
                     Faulting application path: C:\Program
                     Files\WindowsApps\Microsoft.WindowsTerminal_1.11.2921.0_x64__8wekyb3d8bbwe\WindowsTerminal.exe
                     Faulting module path: C:\Program Files\WindowsApps\Microsoft.WindowsTerminal_1.11.2921.0_x64__8wekyb3d8bbwe\Mi
                     crosoft.Terminal.Settings.Editor.dll
                     Report Id: cf3a5110-ac7b-47d1-ad0b-a5db20d997ed
                     Faulting package full name: Microsoft.WindowsTerminal_1.11.2921.0_x64__8wekyb3d8bbwe
                     Faulting package-relative application ID: App
Category           : Application Crashing Events
CategoryNumber     : 100
ReplacementStrings : {WindowsTerminal.exe, 1.11.2110.19001, 616f34a4, Microsoft.Terminal.Settings.Editor.dll...}
Source             : Application Error
TimeGenerated      : 11/6/2021 1:34:08 PM
TimeWritten        : 11/6/2021 1:34:08 PM
UserName
```

After cleaning all settings there were no more crashes when editing actions.

**Comment by zadjii-msft** [comment]
Well, that's pretty trivially reproducible.

**Comment by zadjii-msft** [comment]
Shockingly, I think this has already been fixed in main. I can't repro it on main, so this was fixed even without Controls V2 (#12241). There's been a lot of changes since 1.11.2921, so it'd be hard to pin down which exactly would have fixed it. 

I'm not sure if this is fixed in Preview (1.12.*), but you may have some luck with that version in the meantime.
